.github/workflows/make_release.yml

```diff
@@ -32,13 +32,6 @@ jobs:
           username: ${{ secrets.BINTRAY_USER }}
           password: ${{ secrets.BINTRAY_API_KEY }}
 
-      - name: set up go 1.15
-        uses: actions/setup-go@v1
-        # cause we setup go ourselves otherwise
-        if: ${{ !contains(github.event.ref, 'fips') }}
-        with:
-          go-version: 1.15
-
       - name: Run release builder
```

Comment:
Is setting up regular GO added to create_release script?

Reply:
nope this is a wip